vita3k/gui/src/about_dialog.cpp

```diff
@@ -71,6 +71,10 @@ void draw_about_dialog(GuiState &gui, EmuEnvState &emuenv) {
 
     ImGui::Text("%s", lang["vita3k"].c_str());
     ImGui::Spacing();
+#if defined(__ANDROID__)
+    ImGui::TextWrapped("%s", lang["for_android"].c_str());
+    ImGui::Spacing();
+#endif
```

Comment:
Last time i have put code ,for android my pr have get killed with said: Don't add code for android in the master.

Reply:
we have good reason now,  there is only text here

Reply:
I do agree that maybe this PR is not the best to add the android lines, even more when considering that some of them will be removed when merging the android version. Also it is problematic to add some of these lines to the lang struct because it would make it a lot easier to remove them.
I think you can remove the android part (sorry about that) in this PR and we can talk about it later.

Reply:
okay,i got it